### ISSUE TITLE
Broken reference errors piling up logs

### ISSUE THREAD

**Issue by Zyles** [magento/magento2#26504]
Logs becoming bloated and Sentry going crazy with broken reference errors.

```
$ cat var/log/system.log | grep "Broken reference"
[2020-01-23 03:32:33] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:32:39] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:33:44] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:33:45] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:35:56] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:37:41] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:38:48] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:41:30] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:41:32] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:42:48] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:42:50] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:42:51] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:42:52] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:43:05] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:43:07] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:43:15] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:43:17] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:43:29] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:43:45] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:44:26] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:44:28] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:44:46] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:45:01] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:45:03] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:46:15] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:46:25] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:47:41] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:47:56] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:49:35] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []
[2020-01-23 03:49:37] .WARNING: Broken reference: The element with the "category.view.container" ID wasn't found. Verify the ID and try again. [] []

```

The only occurance of changing "category.view.container." is in our theme:

```
app/design/frontend/Theme/theme/Magento_Theme/layout/default.xml:
<move element="content.top" destination="category.view.container" />
```

### Preconditions (*)

1. Magento 2.3.3 & 2.4-develop
2. Custom theme extended from Luma

### Steps to reproduce (*)

1. Extend luma theme
2. Move the container:

```
app/design/frontend/Theme/theme/Magento_Theme/layout/default.xml:
<move element="content.top" destination="category.view.container" />
```
3. Browse website etc. and check logs after a while

### Expected result (*)

Stop bloating the logs.

### Actual result (*)

Bloating of logs with useless errors.

**Comment by mrtuvn** [comment]
This issue is not related with magento. This is 3rd-party theme issue
The reason is your destination id in custom theme is not exist

**Comment by Zyles** [comment]
Luma is not part of Magento 2?

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Hotel 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-30875` were created

**Issue Available**: @engcom-Hotel, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by mrtuvn** [comment]
> Luma is not part of Magento 2?

I don't say that you should check my reply. I said this problem come from your custom theme

**Comment by mrtuvn** [comment]
<move element="content.top" destination="category.view.container" />
category.view.container only visible in category page
add move directive in default.xml is not best practices

**Comment by magento-engcom-team** [comment]
:x: Something went wrong. Cannot update Jira issue (https://jira.corp.magento.com/browse/MC-30875).

**Comment by magento-engcom-team** [comment]
:x: Something went wrong. Cannot update Jira issue (https://jira.corp.magento.com/browse/MC-30875).

**Comment by magento-engcom-team** [comment]
:x: Something went wrong. Cannot update Jira issue (https://jira.corp.magento.com/browse/MC-30875).

**Comment by magento-engcom-team** [comment]
:x: Something went wrong. Cannot update Jira issue (https://jira.corp.magento.com/browse/MC-30875).

**Comment by magento-engcom-team** [comment]
:x: Something went wrong. Cannot update Jira issue (https://jira.corp.magento.com/browse/MC-30875).

**Comment by magento-engcom-team** [comment]
:x: Something went wrong. Cannot update Jira issue (https://jira.corp.magento.com/browse/MC-30875).

**Comment by slavvka** [comment]
Hi @Zyles. Thank you for your report. 
The issue has been fixed in magento/magento2#28735 by @Bartlomiejsz in 2.4-develop branch
Related commit(s):
 - [9c6653dad7fa63a5c30c25d6c70d93b89dac6ed4](https://github.com/magento/magento2/commit/9c6653dad7fa63a5c30c25d6c70d93b89dac6ed4)

The fix will be available with the upcoming 2.4.1 release.
